Source/santa-driver/SantaDecisionManager.cc

```diff
@@ -29,9 +29,9 @@ bool SantaDecisionManager::init() {
   decision_dataqueue_lock_ = lck_mtx_alloc_init(sdm_lock_grp_, sdm_lock_attr_);
   log_dataqueue_lock_ = lck_mtx_alloc_init(sdm_lock_grp_, sdm_lock_attr_);
 
-  root_decision_cache_ = new SantaCache<uint64_t>(5000, 2);
-  non_root_decision_cache_ = new SantaCache<uint64_t>(500, 2);
-  vnode_pid_map_ = new SantaCache<uint64_t>(2000, 5);
+  root_decision_cache_ = new SantaCache<uint64_t, uint64_t>(5000, 2);
+  non_root_decision_cache_ = new SantaCache<uint64_t, uint64_t>(500, 2);
```

Comment:
Until we fix the issue where system binaries are executing on a non-root VFS we should bump the non_root_decision_cache_ up some.

Reply:
I have a follow up PR that gets rid of the split caches